azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -196,21 +181,13 @@ public Props getProps() {
   }
 
     /**
-     * Checks if a project is active using project_name
-     *
-     * @param name
-     */
-    public Boolean isActiveProject(String name) {
-        return projectsByName.containsKey(name);
-    }
-
-    /**
      * Checks if a project is active using project_id
      *
      * @param name
      */
     public Boolean isActiveProject(int id) {
```

Comment:
It is called in handlePurgeProject. A project is active means the boolean active is set to true in DB for that project. I'll add some comments about it.